docs/repository_structure.md

```diff
@@ -0,0 +1,106 @@
+# SSS Content Repositories
+
+Content for the Security Summer School (SSS) is to developed on GitHub, in the [`security-summer-school` organization](https://github.com/security-summer-school/).
```

Comment:
Typo: "is to developed"

Reply:
Is the format `security-summer-school` needed instead of simply writing **security-summer-school** or security-summer-school? I just think it looks a bit odd:
![image](https://user-images.githubusercontent.com/47995400/137582773-c92ba0d8-6d4f-4409-8802-9894c11cb3f9.png)

Reply:
@carrrina , makes sense, I will remove the formatting.